docs/src/main/sphinx/installation/cli.rst

```diff
@@ -37,6 +37,27 @@ password value to avoid the prompt.
 Use ``--help`` to see information about specifying the keystore, truststore, and
 other authentication details as required. If using Kerberos, see :doc:`/security/cli`.
 
+External authentication - SSO
+-----------------------------
+
+The  ``--external-authentication`` option is used for browser-based SSO
+authentication, as detailed in :doc:`/security/oauth2`. With this configuration,
+the CLI displays an URI that you have to open in a web browser for authentication.
```

Comment:
displays a URL that

Reply:
wow nice catch I didn't know about that.